services/core-web/src/components/noticeOfWork/applications/review/ReviewApplicationFeeContent.tsx

```diff
@@ -172,7 +172,7 @@ export const ReviewApplicationFeeContent: FC<ReviewApplicationFeeContentProps> =
     );
   }, [proposedStartDate, proposedAuthorizationEndDate]);
 
-  useEffect(() => {}, [adjustedTonnage, proposedTonnage]);
+  useEffect(() => { }, [adjustedTonnage, proposedTonnage]);
```

Comment:
Not your doing, but what the hell is this?

Reply:
Ha!  Yeah, I can't wait to get our hands all up in that NOW nonsense.  I know it will be hellish, but lots of it really needs some attention.